src/lib/webClipper/sanitizeHtml.ts

```
@@ -0,0 +1,48 @@
+    "input",
+    "button",
+    "select",
+    "textarea",
+    "[onclick]",
+    "[onerror]",
+    "[onload]",
+  ];
+
+  unwantedSelectors.forEach((selector) => {
+    const elements = div.querySelectorAll(selector);
+    elements.forEach((el) => el.remove());
```

Biome (2.4.7)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)